# .skip.

# Skipping because the error recovery code is not found in Compiler.testifyFilesNamed, but in Compiler.compileFilesNamed.
# Could be fixed...

# .args. -reveal-internal-exceptions=0

# This tests that internal exceptions get wrapped up as an "internal error" and with the correct line number.
# This is useful behavior to end users who don't need or even want an exception report from the Cobra compiler when it has a bug.


%% throw   # .error. internal error

class Program

	def main is shared
		pass
